src/popup/components/Footer.svelte

```diff
@@ -0,0 +1,4 @@
+<div class="tr code f7">
+  <a href="https://github.com" target="_blank" rel="noreferrer">Github</a>
+  ﹣ v1.1.0
```

Comment:
We should update the link and version number here, or we could also drop the footer altogether since it's easy to forget to update this.